docs/reference/configuration.md

```diff
@@ -52,10 +52,10 @@ This reference uses `.` to denote the nesting of values.
     + `database.postgres.connection_max_lifetime` `(duration : 5m)` - Sets the maximum amount of time a connection may be reused `(valid units: ns|us|ms|s|m|h)` 
   + `database.dynamodb` - Configuration section when using `database.type="dynamodb"`
     + `database.dynamodb.table_name` `(string : "kvstore")` - Table used to store the data
-    + `database.dynamodb.read_capacity_units` `(int : 1000)` - Throughput parameter for read capacity
-    + `database.dynamodb.write_capacity_units` `(int : 1000)` - Throughput parameter for write capacity
+    + `database.dynamodb.read_capacity_units` `(int : optional)` - Throughput parameter for read capacity
+    + `database.dynamodb.write_capacity_units` `(int : optional)` - Throughput parameter for write capacity
```

Comment:
Not sure, but isnt the number in parentheses shows the default?

Reply:
Both default and optional/required you can see examples for it in `auth.encrypt.secret_key` `auth.ldap.server_endpoint` `auth.ldap.bind_dn`...
If you think this is confusing we should change this for all the document

Reply:
I agree. If we specify a default-value, it implies that this parameter is optional to the user, right?
I would refrain from using `optional` all together, and just use `required` to show there is no default, and the configuration must include the value, or, the default-value, in case it is optional.